cmd/gen-go-sample/testdata/sample_enum.want

```diff
@@ -34,6 +34,7 @@ func sampleUnaryMethod(myFruit foopb.AType_FruitEnum) error {
 	}
 
 	// myFruit := foopb.AType_APPLE
+
```

Comment:
I'm not sure if I like this line break. I feel like it should be "on top" of the usage of said variable...is it to separate the inline comment from the generated value comments?

Reply:
+1 to Noah's comment.

Reply:
I tend to agree the blank line looks weird here. I was wondering whether when we initialize multiple fields, it would make things more readable to have a blank line, but I'm not sure it will.